net-misc/youtube_dl/youtube_dl-2018.02.08.recipe

```diff
@@ -9,9 +9,9 @@ HOMEPAGE="http://rg3.github.io/youtube-dl/"
 COPYRIGHT="2006-2018 Ricardo Garcia Gonzalez"
 LICENSE="Public Domain"
 REVISION="1"
-SOURCE_URI="https://github.com/rg3/youtube-dl/releases/download/$portVersion/youtube-dl#noarchive"
-CHECKSUM_SHA256="157be71476a97dae99a18df5452472e1786f39b3fe2f968dfa0f81c144d81734"
-SOURCE_FILENAME="youtube-dl-$portVersion.py"
+SOURCE_URI="https://github.com/rg3/youtube-dl/releases/download/$portVersion/youtube-dl-$portVersion.tar.gz"
```

Comment:
Thanks for the info. I had not noticed the full tarball was also available from the upstream website.

That said, I think it is nicer to have an https URI instead of an http URI, even if the latter is shorter than the former.

Reply:
@fbrosson Thanks for the pointer. The upstream site also have https :stuck_out_tongue:

Reply:
Cool! I'll switch `HOMEPAGE` to https!

BTW, your `SOURCE_URI` sends a redirect (`HTTP/1.1 302 Found`) to GitHub's URI.
So we could use any of them.
But since we are getting a "302" (and not a "301"), I'll switch to **_your_** `SOURCE_URI` :)
(Your URI is better, as you already know, because it can save us an update in case upstream ever chooses to switch to another hosting service.)